### ISSUE TITLE
how to pass other parameter in forward

### ISSUE THREAD

**Issue by joshylchen** [sksq96/pytorch-summary#153]
For example in the forward path, I define as
def forward(self, input, path1=True):

I realize that when I want to pass summary(model, [(3, 32,32),(True)]) 
or 
summary(model, [(3, 32,32),(True)] 
or
summary(model, [(3, 32,32),True])
or
summary(model, ((3, 32,32),True))

It either reported as agument must be an iterable or size must be tuple of ints.
How could I resolve this to pass other parameters?

Thank you

**Comment by PANhuihuihuihui** [comment]
you may refer to the original code and adjust it if you want to let this package could pass other parameter. 

> batch_size of 2 for batchnorm
> x = [torch.rand(2, *in_size).type(dtype).to(device=device)
>     for in_size, dtype in zip(input_size, dtypes)]

> model(*x)
